src/client/components/CommentFooter/CommentFooter.less

```diff
@@ -15,7 +15,21 @@
     font-size: 14px;
     margin: 0 8px;
   }
+  
+  &__bullet_br:after {
+      content: "\2022";
+      font-size: 14px;
+      margin: 0 8px;
+    display: block;
+    visibility: hidden;
+    line-height: 0.5;
 
+  @media only screen and (min-width: 468px){
```

Comment:
for the most part this less files is fine aside from a few spacing issues.. it should be more like this:

```
  &__bullet_br:after {
    content: "\2022";
    font-size: 14px;
    margin: 0 8px;
    display: block;
    visibility: hidden;
    line-height: 0.5;

    @media only screen and (min-width: 468px) {
      display: unset;
      visibility: unset;
    }
  }
```